commands/config.py

```diff
@@ -130,6 +131,11 @@ class AnnounceBotConfig(Config):
         "green_tick": "312314752711786497"
     }
 
+    messages = {
```

Comment:
these are added to an existing config, i think we can't share config files between 2 plugins without having to duplicate all properties in both, might need a seperate one